main/ejml-ddense/src/org/ejml/dense/row/CovarianceOps_DDRM.java

```diff
@@ -54,7 +54,7 @@ public static boolean isValidFast( DMatrixRMaj cov ) {
      * @return 0 = is valid 1 = failed positive diagonal, 2 = failed on symmetry, 2 = failed on positive definite
```

Comment:
1 = failed non-negative diagonal

Reply:
and 3 = failed on positive definite

Reply:
Thanks! Fixed and improved documentation.